src/motor-interface-pru.p

```diff
@@ -202,14 +216,30 @@ QUEUE_READ:
 	ZERO &mstate, SIZE(mstate)	; clear the motor states
 	ZERO &r3, 4			; initialize delay calculation state register.
 
+	;; STATUS REGISTER
+	;; ! We are assuming that writing the 4 bytes status register is atomic
```

Comment:
Also we assume that the lower 24 bits are all zero when we are here, because we guaraneed that they are zero at the end of the last step.
Given that we just add values here, this might be useful to mention.